src/shared/hooks/storage/useSavedCourses.ts

```diff
@@ -0,0 +1,53 @@
+import useLocalStorage from './useLocalStorage';
```

Comment:
I think it would make sense to leave this out of the `storage` folder to imply that the other hooks (sessionStorage/localStorage) are hooks to be used to build out other hooks like this one.